src/Games/NexusMods.Games.BethesdaGameStudios/Capabilities/BethesdaFolderMatchInstallerCapability.cs

```diff
@@ -96,8 +96,9 @@ public class BethesdaFolderMatchInstallerCapability : AFolderMatchInstallerCapab
         }
     };
 
-    protected override IEnumerable<InstallFolderTarget> InstallFolderTargets { get; } = new[]
+    /// <inheritdoc />
+    public override IEnumerable<InstallFolderTarget> InstallFolderTargets()
     {
-        GameRootInstallFolderTarget
-    };
+        return new[] { GameRootInstallFolderTarget };
+    }
 }
```

Comment:
Use a static read-only field instead of allocating a new array on every call.

Reply:
Pushed some changes, to only allocate once. What do you think of the new version?